apps/web/scripts/installExternalCourses.py

```diff
@@ -0,0 +1,20 @@
+import json
+import sys
+import os
+
+def main():
+    if len(sys.argv) != 2:
+        exit(f"Usage: {sys.argv[0]} PATH_TO_CONFIG_FILE")
+    filename = sys.argv[1]
```

Comment:
```suggestion
    config_filename = sys.argv[1]
```

I would argue that filename is not a great variable name because it more or less just describes the "type" of the value, not it's purpose or role